### PR TITLE
feat: admin dashboard

### LINKED ISSUE #11: admin dashboard
**Is your feature request related to a problem? Please describe.**
Currently there is no out-of-the-box hosted experience for ReChunk. The only way to visualize and manage chunks is through the CLI.

**Describe the solution you'd like**
Create an admin dashboard where developers can manage current features to manage chunks an future features such as rollbacks etc.

The dashboard should be created using:
- [remix](https://remix.run/)
- [shadcn/ui](https://ui.shadcn.com/)
- [prisma](https://www.prisma.io/)
- [tailwind](https://tailwindcss.com/)
- [ansible](https://www.ansible.com/)
- [sqlite](https://www.sqlite.org/)

This will remove the necessity for the hono server i.e. `/app` and Docker i.e. `Dockerfile` and `.dockerignore`. The admin dashboard can be utiilized as the UI and also API endpoints.

The API endpoints should be updated to match the following details.

```
GET /api/v1/projects/:projectId
POST /api/v1/projects/:projectId
GET /api/v1/projects/:projectId/chunks
POST /api/v1/projects/:projectId/chunks/:chunkId
GET /api/v1/projects/:projectId/chunks/:chunkId
DELETE /api/v1/projects/:projectId/chunks/:chunkId
```

We will use a VPS via Linode and a domain https://rechunk.xyz via Hostinger.

**Describe alternatives you've considered**
Create a simple web app that interacts with the hono server served via docker-compose.

### PR DESCRIPTION
**Pull request by crherman7**
## Scope

[closes TICKET-11](https://github.com/crherman7/rechunk/issues/11)

## Implementation

Utilize [remix.run](https://remix.run/) to bootstrap our admin dashboard. Currently implemented:

- landing page
- dashboard layout
- resource endpoints scaffolding
- database models
- deployment

## Screenshots

<details>
  <summary>ios</summary>

  

Simulator Screenshot - iPhone 15 - 2024-09-29 at 11 48 43
</details>

<details>
  <summary>android</summary>
  
  

Screenshot_1727625072
</details>

<details>
  <summary>web</summary>

  

Screen Shot 2024-09-29 at 11 55 42
Screen Shot 2024-09-29 at 11 55 38
</details>

## How to Test

Run the example app to ensure nothing is broken dependency-wise since this a monorepo. Run the admin site from the `apps/www` directory.

## Emoji Guide

**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**

E.g: Praise, minor suggestions, or clarifying questions that don’t block merging the PR.

> 🟢 Nice refactor!

> 🟡 Why was the default value removed?

E.g: Blocking feedback must be addressed before merging.

> 🔴 This change will break something important

| | | |
| --- | --- | --- |
| Blocking | 🔴 ❌ 🚨 | RED |
| Non-blocking | 🟡 💡 🤔 💭 | Yellow, thinking, etc |
| Praise | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |

### CODE REVIEW

File: apps/www/.eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/**
```
Comment (devs9) [inline apps/www/.eslintrc.cjs:1]: 🤔 @crherman7 what do you think about using the liner from shared package?
and we can extend our linter with a new file for web

for current config I see some errors
✖ 23 problems (23 errors, 0 warnings)

and we can make same updates for `tsconfig.json`

**Review by devs9 — COMMENTED** [review]
💚 Updates for the dashboard looks pretty good! and no problems were found in apps(related to dependency in monorepo) 👍 

but in `www` app, I noticed some errors for `tsc`
<details>
  <summary markdown="span">demo</summary>

  <p align="center">
    
  </p>
</details>

File: apps/www/.eslintrc.cjs (hunk 1 above)
Comment (crherman7) [reply apps/www/.eslintrc.cjs:1]: I think it would be good to extend upon a shared package for eslint and tsconfig likely

Comment (crherman7) [reply apps/www/.eslintrc.cjs:1]: let's do this in another PR

**Pull request merged**